deployment/k8s/README.md

```diff
@@ -49,22 +58,24 @@ The full environment helm charts consists of:
 * Target cluster
 * Migration services
 
-**Note**: For first-time deployments and deployments after changes have been made to a dependent helm package, such as the `migration-console` chart, the following command is needed to update dependent charts
+**Note**: For first-time deployments and deployments after changes have been made to a dependent helm package, such as the `migrationConsole` chart, the following command is needed to update dependent charts
 ```shell
-helm dependency update migration-assistant
+helm dependency update
 ```
 
-The full environment helm chart can be deployed with the helm command
+The full Migration Assistant environment with Mock Customer Clusters can be deployed with the following helm commands
```

Comment:
[Clarity] I could see cases for creating shorthands, but Mock Customer Clusters -> `mcc` is too much for me, can we name this test-clusters or something.

While MigrationAssistant -> `ma` is clearer, and we will use ma quite often in our systems can we avoid the acronym?

Reply:
Personally I like the shorthands to type commands quickly and these both map to their chart names abbreviated so I haven't felt any confusion around them. My anticipation is that normal users will be abstracted away from this naming completely, and its more developers or advanced users that will be playing with these commands.